2020-02-22 11:41:08,328 - instabot version: 0.109.0 (bot) - INFO - Instabot version: 0.109.0 Started
2020-02-22 11:41:08,328 - instabot version: 0.109.0 (bot) - DEBUG - Bot imported from /Users/niels/Documents/instabot/examples/../instabot/bot/bot.py
2020-02-22 11:41:11,081 - instabot version: 0.109.0 (api_login) - DEBUG - Loading cookies
2020-02-22 11:41:11,081 - instabot version: 0.109.0 (api_login) - DEBUG - Loading uuids
2020-02-22 11:41:11,082 - instabot version: 0.109.0 (api_login) - INFO - Recovery from /Users/niels/Documents/instabot/examples/config/dronevlieger_uuid_and_cookie.json: COOKIE True - UUIDs True - TIMING, DEVICE and ...
- user-agent=Instagram 117.0.0.28.123 Android (28/9.0; 420dpi; 1080x1920; OnePlus; ONEPLUS A3003; OnePlus3; qcom; en_US; 180322800)
- phone_id=d936e783-96b0-4684-b21b-2afb58ecfcf7
- uuid=122b7682-5cd8-41d4-a8db-31a23d014db0
- client_session_id=bdef51f8-9da5-43c9-8796-4c66a73ef6f1
- device_id=android-17757404dd6a9a97
2020-02-22 11:41:11,082 - instabot version: 0.109.0 (api_login) - INFO - LOGIN FLOW! Just logged-in: False
2020-02-22 11:41:12,108 - instabot version: 0.109.0 (api) - DEBUG - POST to endpoint: feed/timeline/ returned response: <Response [403]>
2020-02-22 11:41:12,108 - instabot version: 0.109.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "user_has_logged_out", "logout_reason": 9, "status": "fail"}
2020-02-22 11:41:12,108 - instabot version: 0.109.0 (api) - ERROR - Request returns 403 error!
2020-02-22 11:41:13,091 - instabot version: 0.109.0 (api) - DEBUG - POST to endpoint: feed/reels_tray/ returned response: <Response [403]>
2020-02-22 11:41:13,091 - instabot version: 0.109.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "user_has_logged_out", "logout_reason": 9, "status": "fail"}
2020-02-22 11:41:13,091 - instabot version: 0.109.0 (api) - ERROR - Request returns 403 error!
2020-02-22 11:41:13,092 - instabot version: 0.109.0 (api) - INFO - Login flow failed, the cookie is broken. Relogin again.
2020-02-22 11:41:13,138 - instabot version: 0.109.0 (api_login) - DEBUG - Loading cookies
2020-02-22 11:41:13,138 - instabot version: 0.109.0 (api_login) - DEBUG - Loading uuids
2020-02-22 11:41:13,138 - instabot version: 0.109.0 (api_login) - INFO - Recovery from /Users/niels/Documents/instabot/examples/config/dronevlieger_uuid_and_cookie.json: COOKIE True - UUIDs True - TIMING, DEVICE and ...
- user-agent=Instagram 117.0.0.28.123 Android (28/9.0; 420dpi; 1080x1920; OnePlus; ONEPLUS A3003; OnePlus3; qcom; en_US; 180322800)
- phone_id=d936e783-96b0-4684-b21b-2afb58ecfcf7
- uuid=122b7682-5cd8-41d4-a8db-31a23d014db0
- client_session_id=bdef51f8-9da5-43c9-8796-4c66a73ef6f1
- device_id=android-17757404dd6a9a97
2020-02-22 11:41:13,138 - instabot version: 0.109.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2020-02-22 11:41:14,130 - instabot version: 0.109.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2020-02-22 11:41:15,242 - instabot version: 0.109.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2020-02-22 11:41:16,244 - instabot version: 0.109.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2020-02-22 11:41:17,229 - instabot version: 0.109.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2020-02-22 11:41:18,216 - instabot version: 0.109.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2020-02-22 11:41:19,208 - instabot version: 0.109.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2020-02-22 11:41:20,325 - instabot version: 0.109.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2020-02-22 11:41:23,161 - instabot version: 0.109.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [200]>
2020-02-22 11:41:23,161 - instabot version: 0.109.0 (api) - INFO - Logged-in successfully as 'dronevlieger'!
2020-02-22 11:41:23,161 - instabot version: 0.109.0 (api_login) - INFO - LOGIN FLOW! Just logged-in: True
2020-02-22 11:41:24,269 - instabot version: 0.109.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2020-02-22 11:41:25,246 - instabot version: 0.109.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2020-02-22 11:41:26,209 - instabot version: 0.109.0 (api) - DEBUG - GET to endpoint: zr/token/result/?device_id=android-17757404dd6a9a97&token_hash=&custom_device_id=android-17757404dd6a9a97&fetch_reason=token_expired returned response: <Response [200]>
2020-02-22 11:41:27,177 - instabot version: 0.109.0 (api) - DEBUG - GET to endpoint: multiple_accounts/get_account_family/ returned response: <Response [200]>
2020-02-22 11:41:28,278 - instabot version: 0.109.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2020-02-22 11:41:29,241 - instabot version: 0.109.0 (api) - DEBUG - GET to endpoint: banyan/banyan/?views=['story_share_sheet','threads_people_picker','group_stories_share_sheet','reshare_share_sheet'] returned response: <Response [400]>
2020-02-22 11:41:29,241 - instabot version: 0.109.0 (api) - DEBUG - Responsecode indicates error; response content: {"message": "Bad request", "status": "fail"}
2020-02-22 11:41:29,241 - instabot version: 0.109.0 (api) - ERROR - Request returns 400 error!
2020-02-22 11:41:29,241 - instabot version: 0.109.0 (api) - INFO - Instagram's error message: Bad request
2020-02-22 11:41:29,241 - instabot version: 0.109.0 (api) - INFO - Instagram's error message: {}
2020-02-22 11:41:29,242 - instabot version: 0.109.0 (api_login) - ERROR - Exception raised: 'API' object has no attribute 'igtv_browse_feed'
Traceback (most recent call last):
  File "/Users/niels/Documents/instabot/examples/../instabot/api/api_login.py", line 151, in login_flow
    check_flow.append(self.igtv_browse_feed())
AttributeError: 'API' object has no attribute 'igtv_browse_feed'

2020-02-22 11:41:29,242 - instabot version: 0.109.0 (bot_checkpoint) - DEBUG - Loading Checkpoint file from: /Users/niels/Documents/instabot/examples/config/dronevlieger.checkpoint
2020-02-22 11:43:23,222 - instabot version: 0.109.0 (api) - DEBUG - POST to endpoint: accounts/logout/ returned response: <Response [200]>
2020-02-22 11:43:23,225 - instabot version: 0.109.0 (bot) - INFO - Bot stopped. Worked: 1:04:05.311680
2020-02-22 11:43:23,225 - instabot version: 0.109.0 (bot_checkpoint) - DEBUG - Saving Checkpoint file to: /Users/niels/Documents/instabot/examples/config/dronevlieger.checkpoint
2020-02-22 11:43:23,226 - instabot version: 0.109.0 (bot) - INFO - Total requests: 82
2020-02-22 11:43:23,226 - instabot version: 0.109.0 (bot_checkpoint) - DEBUG - Saving Checkpoint file to: /Users/niels/Documents/instabot/examples/config/dronevlieger.checkpoint
2020-02-22 11:43:23,227 - instabot version: 0.109.0 (bot) - INFO - Total requests: 82
